### PR TITLE
Fix log level and AMQP bridge

### PR DESCRIPTION
**Pull request by ppatierno**
This PR set the INFO log level for the bridge by default (instead of the current DEBUG one) and also disable the AMQP bridge by default.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
+dist: xenial
 sudo: required
 language: java
 jdk:
-- oraclejdk8
-- oraclejdk11
+- openjdk8
+- openjdk11
```
Comment (stanlyDoge) [inline .travis.yml]: Should this be in another PR?

Comment (ppatierno) [reply .travis.yml]: could be but it fixes the current broken build

Comment (stanlyDoge) [reply .travis.yml]: Ok
